examples/lines_bars_and_markers/fill.py

```diff
@@ -20,7 +20,7 @@ def koch_snowflake(order, scale=10):
     """
     Return two lists x, y of point coordinates of the Koch snowflake.
 
-    Arguments
+    Parameters
     ---------
```

Comment:
Number of dashes should match the length of the caption.

Reply:
You are right, I just added the missing dash. Thanks!